html_css_exercise/femc_level1_huddleLandingPage/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="css/index.css" />
+        <link rel="stylesheet" href="css/reset.css" />
+        <title>femc-level1</title>
+        <link rel="icon" href="img/favicon-32x32.png" type="image/x-icon" />
+    </head>
+    <body>
+        <header class="header-main">
+            <a href="index.html">
+                <img src="img/logo.svg" alt="huddle-logo" />
+            </a>
+        </header>
+
+        <main class="container-main">
+            <div class="main-img-mockup">
+                <img src="img/illustration-mockups.svg" alt="mockups-image" />
+            </div>
+            <div class="main-message">
+                <h2 class="main-h2-title">
```

Comment:
Headers should always maintain a hierarchy on the page. An h2 should not be used before an h1. Styling is irrelevant. Heading levels give the browser much needed context for accessibility. Pages should always have one h1, then h2 h3 ... h6 (each level should be more deeply nested in the DOM than the last).

Try not to include markup in class names. Classes should be reusable styles independent of the tags they are used on. 

```suggestion
                <h1 class="main-title">
```